include/deal.II/fe/fe_nedelec.h

```diff
@@ -324,6 +324,18 @@ class FE_Nedelec : public FE_PolyTensor<dim>
   virtual std::unique_ptr<FiniteElement<dim, dim>>
   clone() const override;
 
+  /**
+   * For a finite element of degree @p sub_degree < @p degree, we return a
+   * vector which maps the numbering on an FE of degree @p sub_degree into the
+   * numbering on this element.
+   *
+   * Note that for the Nedelec element, by @p sub_degree,
+   * we refer to the maximal polynomial degree (in any coordinate direction) as
+   * opposed to the Nedelec degree.
+   */
+  virtual std::vector<unsigned int>
+  get_embedding_dofs(const unsigned int sub_degree) const override;
```

Comment:
Even for the Nedelec element, isn't this only applicable for, say, element orders 1 and n, 2 and 2n, 3 and 3n (assuming we are using equispaced points), etc?

Reply:
Perhaps I have misunderstood portions of the Nedelec element implementation in deal.II, but if this were the case I would have expected that this would have been indicated by the test I wrote. I also tested beforehand with the original way I was mapping from p-reduced Nedelec cells to enriched ones using the curl-conforming projection-based interpolation.

I'll have to dive back into the `FE_Nedelec` implementation and see if there is something I missed that might prevent identification of equivalent dofs between a Nedelec cell of `sub_degree` and a Nedelec cell of `sup_degree` in this manner for some cases.

Reply:
Think about the usual Q_k elements in 2d. If you have a Q_2, it has DoFs at the vertices, edge midpoints, and cell midpoint. If you have a Q_3, it has DoFs at the vertices but also at positions 1/3 and 2/3 along the edge, and 4 in the interior of the cell which are all not located at the cell center. I don't think the Nedelec element has a structure that is different from the general problem outlined above unless it is *moment based* along edges in which all edge and cell degrees of freedom might be located at edge and cell midpoints.

Reply:
The case for `FE_Nedelec` is not quite the same as the standard Q_k element. The support points are the Gauss-Legendre quadrature points (on the faces or the interior of the cell), but the identification of equivalent DoFs from a Nedelec element of `sub_degree` to an element of `sup_degree` is based on the hierarchical construction of the mixed order Nedelec polynomials as in `PolynomialsNedelec<dim>::create_polynomials()`. In other words, the enriched cell includes all the shape functions of the reduced cell, so there is no concern w.r.t. the choice of `sub_degree` or `sup_degree`.